mongo/migrate/db.go

```diff
@@ -64,3 +65,23 @@ func UpdateMigrationInfo(version Version, sess *mgo.Session, db string) error {
 
 	return nil
 }
+
+func GetTenantDbs(sess *mgo.Session, baseDb string) ([]string, error) {
+	s := sess.Copy()
+	defer s.Close()
+
+	dbs, err := s.DatabaseNames()
+	if err != nil {
+		return nil, err
+	}
+
+	prefix := baseDb + "-"
```

Comment:
I'm not quite convinced about this. The only place where the code is tenant aware is `go-lib-micro/store` package. I think I would prefer if `GetTenantDbs` would take a `func TenantDbMatcher(name string) bool` as a parameter. Then define a func in `store`:

```
type TenantDbMatchFunc func (name string) bool

func IsTenantDb(baseDb string) TenantDbMatchFunc {
    prefix := baseDb + "-"
    return func (name string) bool {
        return strings.HasPrefix(name, prefix)
    }
}
```

Then the code would read:

```
migrate.GetTenantDbs(session, store.IsTenantDb(someprefix))
```